mythril/analysis/templates/callgraph.html

```diff
@@ -8,17 +8,22 @@
     {% if not phrackify %}
     <style type="text/css">
         #mynetwork {
+          height: 100%;
             background-color: #232625;
         }
         body {
             background-color: #232625;
             color: #ffffff;
             font-size: 10px;
         }
+        html, body {
```

Comment:
These can be moved to outside the if else, but in this case it might not affect that much as it's just 1 condition.